src/NuGet.Core/NuGet.Packaging/PackageExtraction/StreamExtensions.cs

```diff
@@ -5,12 +5,20 @@
 using System.Buffers;
 using System.IO;
 using System.IO.MemoryMappedFiles;
+using System.Threading.Tasks;
+using System.Threading;
 using NuGet.Common;
 
 namespace NuGet.Packaging
 {
     public static class StreamExtensions
     {
+        public static readonly int BufferSize = 8192;
```

Comment:
Please refer to https://github.com/NuGet/NuGet.Client/blob/dev/docs/nuget-sdk.md#changes-to-apis. 

We should be mindful about adding public types and methods. 
Is there a benefit to making buffer size public?